CONTRIBUTING.md

```diff
@@ -80,17 +80,18 @@ Please contribute! The Layer5 site uses Gatsby. The process of contributing to d
 1. Get a local clone of your fork of the site.
 `git clone https://github.com/layer5io/layer5`
 1. Switch to the `master` branch.
-`git checkout master`
+`git switch master`
 1. Create and checkout a new branch to make changes within
-`git checkout -b <my-username>/<my-changes>`
+`git checkout -b <my-changes-branchname>`
 1. Edit site/add content.
 `vi <specific page>.md` # or use your favorite IDE
 1. Run site locally to preview changes.
 `make site` # this will run a local web server with "live reload" conveniently enabled.
+(NOTE make is a linux command , if you are using windows you can install wsl , and then run command `sudo apt install make` )
```

Comment:
I doubt that!
`make` do work in windows as well.
 We have GNU make for Windows.